cmd/compose.go

```diff
@@ -101,6 +101,7 @@ single config file or in separate command instances using different config files
 Specify config location when executing the command:
 ./vulcanizedb compose --config=./environments/config_name.toml`,
 	Run: func(cmd *cobra.Command, args []string) {
+		subCommand = cmd.CalledAs()
```

Comment:
Is it possible to define this helper and drop the explicit `subCommand` field in the rest?

```go
logWithCommand = log.WithField("subCommand", subCommand)
logWithCommand.Fatal(err)
```

Reply:
if this works, setting this globally and using when logging would be dank